doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_cartodb_sink.md

```diff
@@ -64,7 +64,9 @@ The name of these tables depends on the configured data model and analysis mode
 
 The above applies both if `enable_raw` or `enable_distance` es set to `true`. In addition, the distance analysis mode adds the sufix `x0000distance` to the table name.
 
-It must be said [PostgreSQL only accepts](https://www.postgresql.org/docs/current/static/sql-syntax-lexical.html#SQL-SYNTAX-IDENTIFIERS) alphanumeric characters and the underscore (`_`). Remember certain [encoding](#section2.3.4) is applied.
+Since based in [PostgreSQL](https://www.postgresql.org/docs/current/static/sql-syntax-lexical.html#SQL-SYNTAX-IDENTIFIERS), it must be said only alphanumeric characters and the underscore (`_`) are accepted.. This leads to certain [encoding](#section2.3.4) is applied.
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/pull/1137/commits/214a3500e081260edff1820df94bd1a43dd21669